### ISSUE TITLE
UI reporting - error when Clicking in rule

### ISSUE THREAD

**Issue by tpurschke** [CactuseSecurity/firewall-orchestrator#1340]
Report – error found with latest develop release (https://github.com/CactuseSecurity/firewall-orchestrator/pull/1339)
```
==> /var/log/fworch/ui.log <==
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]: Error - Object Fetch (MainLayout.razor in line 116): Unspecified error occured:
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]:  ---
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]: Exception thrown:
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]:  NullReferenceException
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]: Message:
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]:  Object reference not set to an instance of an object.
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]: Stack Trace:
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]:  at FWO.Ui.Shared.ObjectGroup`1.<>c__DisplayClass59_0.<FetchContent>b__1(Management[] managementsUpdate) in /usr/local/fworch/ui/files/FWO.UI/Shared/ObjectGroup.razor:line 325
Dec  6 17:32:35 orctstfits202 fworch-ui[378250]:    at FWO.Ui.Pages.Reporting.Report.FetchContent(Byte contentType, Func`2 callback, Int64 id) in /usr/local/fworch/ui/files/FWO.UI/Pages/Reporting/Report.razor:line 324
```

**Comment by NilsPur** [comment]
Null check wasnt correct @Y4nnikH

**Comment by Y4nnikH** [comment]
Oh yeah I see, thanks for fixing it!
Now it should work correctly, although I'm wondering what caused the management object to not have the fetched rule info in that situation..

**Comment by tpurschke** [comment]
@NilsPur please create PR for this issue from your fork

**Comment by tpurschke** [comment]
fixed in #1357
